systemd/cloud-init-hotplugd.service

```diff
@@ -12,6 +12,7 @@
 [Unit]
 Description=cloud-init hotplug hook daemon
 After=cloud-init-hotplugd.socket
+After=cloud-init.target
```

Comment:
This should be `cloud-final.servce`, correct?

Reply:
I think it is equivalent, as `cloud-init.target` is a sync point after `cloud-config.service` and `cloud-final.service`, but more resilient for future changes.

But I am good either way.